assets/js/vendor/private-eye-1.0.0.js

```diff
@@ -0,0 +1,24 @@
+// https://github.com/18F/private-eye
+(function() {
+  'use strict';
+
+  var PrivateEye = function(opts) {
+    var styles = document.createElement('style');
+    styles.innerHTML = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
+    document.body.appendChild(styles);
+
+    opts.ignoreUrls.forEach(function(url) {
+      var anchors = document.querySelectorAll('a[href*="' + url + '"]');
+      Array.prototype.forEach.call(anchors, function(anchor) {
+        anchor.className += ' private-link';
+        anchor.title = "This is a link to a private site, which may or may not be accessible to you.";
```

Comment:
Yep, fixed by https://github.com/18F/private-eye/pull/11, which is included in Private Eye v1.1.0. Adding the old version exactly as it was.